src/insights/position.ts

```diff
@@ -21,4 +22,100 @@ export interface LeaderboardResult {
   meta: PageMeta;
 }
 
-export type QueryGetLeaderboardResponse = Entries<LeaderboardResult>
\ No newline at end of file
+export type QueryGetLeaderboardResponse = Entries<LeaderboardResult>
+
+//Liquidation And ADL
+export interface GetLiquidationAndADLQueryParams extends QueryByPageRequest {
+  sort?: "ASC" | "DESC";
+}
+
+export interface QueryGetLiquidationAndADLResponse {
+  entries: {
+    id:                     string;
+    blockHeight:            string;
+    triggeredBlockHeight:   string;
+    address:                string;
+    market:                 string;
+    side:                   string;
+    price:                  string;
+    quantity:               string;
+    available:              string;
+    filled:                 string;
+    allocatedMarginAmount:  string;
+    allocatedMarginDenom:   string;
+    status:                 string;
+    orderType:              string;
+    initiator:              string;
+    timeInForce:            string;
+    stopPrice:              string;
+    triggerType:            string;
+    isLiquidation:          boolean;
+    isPostOnly:             boolean;
+    isReduceOnly:           boolean;
+    blockCreatedAt:         Date;
+    poolID:                 string;
+    avgFilledPrice:         string;
+    lastUpdatedBlockHeight: string;
+  }[]
+  meta: PageMeta
+}
+
+//Liquidation Engine
+export interface GetLiquidationEngineParams extends QueryByPageRequest {
+  sort?: "ASC" | "DESC";
+}
+
+export interface QueryGetLiquidationEngineResponse {
+  entries: {
+    id:                     string;
+    blockHeight:            string;
+    triggeredBlockHeight:   string;
+    address:                string;
+    market:                 string;
+    side:                   string;
+    price:                  string;
+    quantity:               string;
+    available:              string;
+    filled:                 string;
+    allocatedMarginAmount:  string;
+    allocatedMarginDenom:   string;
+    status:                 string;
+    orderType:              string;
+    initiator:              string;
+    timeInForce:            string;
+    stopPrice:              string;
+    triggerType:            string;
+    isLiquidation:          boolean;
+    isPostOnly:             boolean;
+    isReduceOnly:           boolean;
+    blockCreatedAt:         Date;
+    poolID:                 string;
+    avgFilledPrice:         string;
+    lastUpdatedBlockHeight: string;
+  }[]
+  meta: PageMeta
+}
+
+//position view | profit size risk
+export interface GetPositionsViewPathParams {
+  view: "risk" | "profit" | "size"
```

Comment:
Maybe for this, you can create and export an enum so that users can input the enum rather than the string